README.md

```diff
@@ -12,3 +12,8 @@ import { hotjar } from 'react-hotjar';
 
 hotjar.initialize(hjid, hjsv);
 ```
+- hjid: Stands for 'Hotjar ID' - Your site's ID. This is the ID which tells Hotjar which site settings it should load and where it should save the data collected.
+
+- hjsv: Stands for 'Hotjar Snippet Version' - The version of the Tracking Code you are using. This is only needed if Hotjar ever updates the Tracking Code and needs to discontinue older ones. Knowing which version your site includes allows us to contact you and inform you accordingly.
```

Comment:
I would say Knowing which version your site includes allows hotjar team contact you and inform you accordingly. instead of Knowing which version your site includes allows us to contact you and inform you accordingly.

Reply:
Thanks for the suggestion! I've changed it in another commit.